app_pihole.md

```diff
@@ -1,5 +1,74 @@
-===== Pihole =====
+# <img src="/images/pihole_logo.png" width="80px" alt="Pi-hole's logo"> Pi-hole
 
-Homepage: https://pi-hole.net
+[![Install Pi-hole with YunoHost](https://install-app.yunohost.org/install-with-yunohost.png)](https://install-app.yunohost.org/?app=pihole) [![Integration level](https://dash.yunohost.org/integration/pihole.svg)](https://dash.yunohost.org/appci/app/pihole)
 
-**Pi-hole®** Network-wide ad blocking via your own DNS server, with nice Performance And Statistics web page.
+### Index
+
+- [Using Pi-Hole as a DHCP server](#using-pi-hole-as-a-dhcp-server)
+  - [Configure Pi-Hole](#configure-pi-hole)
+  - [Configure my router](#configure-my-router)
+  - [Restore Network](#restore-network)
+- [Useful links](#useful-links)
+
+Pi-hole is a network-level ad blocker that acts as a DNS liar and possibly a DHCP3 server for use on a private network. It is designed to be installed on embedded devices with network capabilities, such as the Raspberry Pi, but can be used on other machines running Linux or in virtualized environments.
+
+## Using Pi-Hole as a DHCP server
+
+> **Warning, you should be aware that touching your DHCP could break your network.
+In case your server is inaccessible, you will lose your dns resolution and IP address.
+Thus, you would lose any connection to the internet and even the connection to your router.**
+
+> If you encounter this kind of problem, please read the section "How to restore my network".
+
+### Configure Pi-Hole
+
+There are 2 ways to configure Pi-hole to be used as your DHCP server.
+- Either you can choose to use it when you install the application.
+- Or you can activate the DHCP server afterwards in the "Settings" tab, part "Pi-hole DHCP Server".
+In this second case, it may be preferable to force the server ip to a static address.
+
+### Configure my router
+
+Your router or your ISP's router has a DHCP server enabled by default.
+If you keep this DHCP, along with Pi-hole's, you will have transparent conflicts between them.
+The first DHCP server to respond will distribute its own ip and settings.
+So you need to turn off your router's DHCP server and let Pi-hole manage your network.
+
+#### Why should I use Pi-hole's DHCP?
+
+By using Pi-hole's DHCP, you allow Pi-hole to give its dns configuration to each of your clients. This way, every request will be filtered by Pi-hole.
+
+Another case of using Pi-hole DHCP is if you have hairpinning problems (you can't connect to your server because its ip is your public ip, and your router doesn't allow this).
```

Comment:
```suggestion
Another case of using Pi-hole DHCP is if you have hairpinning problems (you can't connect to your server because its IP is your public IP, and your router doesn't allow this).
```